readme.md

```diff
@@ -113,6 +113,8 @@ Examples:
   lighthouse <url> --quiet --chrome-flags="--headless"                      Launch Headless Chrome, turn off logging
   lighthouse <url> --extra-headers "{\"Cookie\":\"monster=blue\"}"          Stringify\'d JSON HTTP Header key/value pairs to send in requests
   lighthouse <url> --extra-headers=./path/to/file.json                      Path to JSON file of HTTP Header key/value pairs to send in requests
+  lighthouse <url> --only-categories=performance,pwa                        Only run specific categories. Valid categories: accessibility,
```

Comment:
this is periodically updated from the output of `lighthouse --help` do you think you could add the description to the actual cli-flags.js file itself? :)

Reply:
Done. Sorry I should have figured that out. Thanks for your patience.